### DISCUSSION TITLE
Node-RED under Home Assistant hass.io version 5.8

### DISCUSSION THREAD

**Discussion by egenu** [zachowj/node-red-contrib-home-assistant-websocket#315] [Q&A]
Dear all,
I had used Node-RED successfully in the past, but since I reconstructed my Home Assistant hass.io version 5.8 into a PI4/8Gb I am not able to install it from  "Supervisor" => "Add-on Store", maybe lost his "official status"?
Now I had to first install it trough HACS, then inside "Configuration" => Integration to add it.
The issue is that there isn't anymore the Node-RED editor accessible. Could someone give me a hint?
Regards.

**Comment by zachowj** [comment]
I only run HA core but as far as I know, the Node-RED addon is still available, this package is included in the add-on and is not the add-on. The Node-RED that shows up in HACS is a custom integration. There's a difference between integration and an add-on. The custom integration is a companion to this package that lets you do things like create entities and webhooks in HA that trigger flows in NR.

**Reply by egenu** [reply]
Thanks, this makes me found the solution.
In fact , I only had inside my "Add-on Store" the "Official add-ons" and the Node-RED is under the "Home Assistant Community Add-ons"
After installing this one and a restart  the "OPEN WEB UI" is now working back!
